monitor/kafka.go

```diff
@@ -35,6 +35,23 @@ type GatewayEvent struct {
 	Data      interface{} `json:"data"`
 }
 
+type PipelineStatus struct {
```

Comment:
Hmm, maybe I should auto-generate this type for you. Let me see if I can figure that out and will let you know, otherwise let's just keep this